h/test/sidebar_app_config_test.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from h import sidebar_app_config
+
+
+@pytest.mark.parametrize("url_in,url_out", [
+    ("http://example.com", "ws://example.com"),
+    ("https://example.com", "wss://example.com"),
+    ("http://monkey.com:80/foo", "ws://monkey.com:80/foo"),
+    ("https://giraffe.com:1443/ws?hello", "wss://giraffe.com:1443/ws?hello"),
+])
+def test_websocketize(url_in, url_out):
+    result = sidebar_app_config._websocketize(url_in)
```

Comment:
As I've said many times before, please let's not test private functions. Test `app_config` here, not `websocketize`, as there are no external callers of `websocketize` so it's perfectly possible to change the implementation of `app_config` in a way that ensures the app still works correctly, but the tests break, or vice versa.